util/misc-helpers.ts

```diff
@@ -0,0 +1,59 @@
+import { Routes } from '~/enums/routes'
+
+const LOGOUT_URL = 'LOGOUT_URL'
+const LOGIN_URL = 'LOGIN_URL'
+const SITEMINDER_LOGOUT_URL = 'SITEMINDER_LOGOUT_URL'
+const REGISTRY_HOME_URL = 'REGISTRY_HOME_URL'
+
+
+/** Returns true if current route is Signin. */
+export function isSigninRoute (path = window.location.pathname): boolean {
+  return path.startsWith(Routes.SIGNIN)
+}
+
+/** Returns true if current route is Signout. */
+export function isSignoutRoute (path = window.location.pathname): boolean {
+  return path.startsWith(Routes.SIGNOUT)
+}
+
+/** Returns true if current route is Login. */
+export function isLoginRoute (path = window.location.pathname): boolean {
+  return path.startsWith(Routes.LOGIN)
+}
+
+/** "Sleeps" for specified timeout. Must be awaited. */
+ export function sleep (ms: number): Promise<void> {
+  return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+/** Sets URL to return to when user logs out. */
+export function setLogoutUrl (url: string): void {
+  sessionStorage.setItem(LOGOUT_URL, url)
```

Comment:
Do you think these could go into the store instead? What do you think?

Reply:
Yes, a separate store for this could be made, but not entirely required. Since we can make individual stores and use them as needed in pinia, as opposed to one giant store in vuex, making this a store would essentially just involve changing the file directory. I was using the BC registry structure for this originally: 

https://github.com/bcgov/bcregistry/blob/9d49276ecab56c25bc7f60bd2a37101e392363e3/utils/misc-helpers.ts#L4

Let me know if you think making this a store is a better idea.

Reply:
I'll leave it up to you. In the past we've used session storage for keys like this, instead of the store. Do whatever you think is most clean and clear.